maat-court-data-api/build.gradle

```diff
@@ -129,6 +129,8 @@ dependencies {
 
     annotationProcessor group: 'org.hibernate', name: 'hibernate-jpamodelgen', version: versions.hibernateJpaModelGen
 
+    implementation 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
Do we need new lib?

Reply:
I think @muthusubra  added this to remove the warning that we get while building. Warning is caused due to the org.springframework.lang.NonNull in MicrometerCloudwatchConfig.java. I fixed it in this PR - LCAM-1097 still in peer review. We can remove this additional jar - com.google.code.findbugs:jsr305:3.0.2

Reply:
Thanks @skasthuri36326 Will remove it